docs/api-designer/getting-started-api-designer/README.adoc

```diff
@@ -115,16 +114,16 @@ Use the {product-long-api-designer} web console to create an OpenAPI or AsyncAPI
 
 ifndef::qs[]
 .Prerequisites
-* You're logged in to the {product-api-designer} web console at {service-url-api-designer}[^].
+* You're logged in to the {product-long-api-designer} web console at {service-url-api-designer}[^].
 endif::[]
 
 .Procedure
-. In the {service-url-api-designer}[{product-api-designer} web console], click *Create a schema or API design*.
+. In the {service-url-api-designer}[{product-long-api-designer} web console^], click *Create a schema or API design*.
 . Complete the form to provide the details of the new design:
 .. *Name*: Enter a unique design name, such as `my-api-design`.
-.. *Summary*: Optional: Enter a short description of the design.
+.. *Summary*: _Optional:_ Enter a short description of the design.
```

Comment:
I realize this might be purely be a matter of personal taste, but there's something about the double colon and use of italics here that feels a little "off". Would either of the following work?

- ` *Summary* (Optional):`
- ` (Optional) *Summary*`

As I said, I realize I might be just expressing my own taste, so feel free to disregard!

Reply:
Ah, I had actually intended to change it to your first suggested option, but forgot! Thanks for the reminder, John!

Reply:
Done